lib/configfiles/leap_42.1.xml

```diff
@@ -0,0 +1,1474 @@
+var.socket_dir  = home_dir + "/sockets"
+
+include "modules.conf"
+server.port = 80
+server.use-ipv6 = "enable"
+server.username  = "lighttpd"
+server.groupname = "lighttpd"
+
+server.document-root = server_root + "/htdocs"
+server.pid-file = state_dir + "/lighttpd.pid"
+server.errorlog             = log_root + "/error.log"
+include "conf.d/access_log.conf"
+include "conf.d/debug.conf"
+server.event-handler = "linux-sysepoll"
+server.network-backend = "sendfile"
+server.max-fds = 2048
+server.stat-cache-engine = "simple"
+server.max-connections = 1024
+
+index-file.names += (
+  "index.xhtml", "index.html", "index.htm", "default.htm", "index.php"
+)
+
+
+
+server.indexfiles    = ("index.php", "index.html",
+						"index.htm", "default.htm")
+						
+server.name			 = "<SERVERNAME>"
+server.port          = 80
+server.bind          = "<SERVERIP>"
+url.access-deny = ("~", ".inc")
+
+fastcgi.server = (
+".php" => (
+		"localhost" => (
+			"socket" => "/tmp/lighttpd-fcgi-sock-lighttpd",
+			"broken-scriptfilename" => "enable",
+			"bin-path" => "/usr/bin/php5-cgi",
+			"min-procs" => 1,
+			"max-procs" => 1,
+			"max-load-per-proc" => 4,
+			"idle-timeout" => 60,
+			"bin-environment" => (
+				"UID" => "wwwrun",
+				"GID" => "www",
+				"PHP_FCGI_CHILDREN" => "0",
+				"PHP_FCGI_MAX_REQUESTS" => "10000"
+			),
+			"bin-copy-environment" => ( "" )
+		)
+	)
+)
+
+#### external configuration files
+## mimetype mapping
+#include_shell "/usr/share/lighttpd/create-mime.assign.pl"
+
+chown lighttpd:lighttpd /var/log/lighttpd/
+
+]]>
+						</content>
+					</file>
+					<include>//service[@type='http']/general/commands</include>
+
+					<command>
+						<visibility mode="isdir">{{settings.system.apacheconf_vhost}}
+						</visibility>
+						<content><![CDATA[echo -e 'include_shell "cat {{settings.system.apacheconf_vhost}}*.conf"' >> /etc/lighttpd/lighttpd.conf]]></content>
+					</command>
+					<command>
+						<visibility mode="isfile">{{settings.system.apacheconf_vhost}}
+						</visibility>
+						<content><![CDATA[echo -e 'include "{{settings.system.apacheconf_vhost}}"' >> /etc/lighttpd/lighttpd.conf]]></content>
+					</command>
+					<command>
+						<visibility mode="isdir">{{settings.system.apacheconf_diroptions}}
+						</visibility>
+						<content><![CDATA[echo -e 'include_shell "cat {{settings.system.apacheconf_diroptions}}*.conf"' >> /etc/lighttpd/lighttpd.conf]]></content>
+					</command>
+					<command>
+						<visibility mode="isfile">{{settings.system.apacheconf_diroptions}}
+						</visibility>
+						<content><![CDATA[echo -e 'include "{{settings.system.apacheconf_diroptions}}"' >> /etc/lighttpd/lighttpd.conf]]></content>
+					</command>
+					<command><![CDATA[lighty-disable-mod cgi]]></command>
+					<command><![CDATA[lighty-disable-mod fastcgi]]></command>
+					<command><![CDATA[service lighttpd restart]]></command>
+				</daemon>
+				<!-- HTTP Nginx -->
+				<daemon name="nginx" title="nginx">
```

Comment:
You are missing the complete `<file>` block for the acme.conf, right before `<file name="/etc/nginx/fastcgi_params">` (Line 214) add: 

`					<file name="/etc/nginx/acme.conf">
						<visibility mode="true">{{settings.system.leenabled}}
						</visibility>
						<content><![CDATA[
location /.well-known/acme-challenge {
	alias {{settings.system.letsencryptchallengepath}}/.well-known/acme-challenge;
	location ~ /.well-known/acme-challenge/(.*) {
		default_type text/plain;
	}
}
]]>
						</content>
					</file>`